Assets/Scripts/ssjAug2018/Data/MailboxData.cs

```diff
@@ -70,5 +70,20 @@ public sealed class MailboxData : ScriptableObject
 
         public int MaxLettersPerBox => _maxLettersPerBox;
 #endregion
+
+        [Space(10)]
+
+#region Audio
+        [Header("Audio files")]
```

Comment:
Instead of doing this, add an EffectTrigger to the Mailbox for each effect that way we can also trigger VFX or animations or whatever with it